concepts/headers-includes/about.md

```diff
@@ -0,0 +1,92 @@
+# About
+
+C requires that all functions are declared _before_ they are defined.
```

Comment:
That's not true. It requires functions to be declared before they are *called* or otherwise used in an expression.  
There is however the compiler option `-Wmissing-declarations` that warns about non-`static`(!) functions that are defined without being previously declared. The idea is that there are typically two kinds of functions, those declared in a `.h` file and meant to be called from other `.c` files, and local (`static`) helper functions that are effectively hidden from other translation units. The latter can be defined without a prior declaration and `-Wmissing-declarations` will not complain.

How about "called" instead of "defined"?